src/js/intl/strings.js

```diff
@@ -713,7 +712,7 @@ exports.strings = {
     'es_AR': 'No podés hacer reset en el modo detached. Usá checkout si querés moverte',
     'es_MX': 'No puedes hacer reset en el modo separado. Usa checkout si quieres moverte',
     'es_ES': 'No puedes hacer reset en el modo separado. Usa checkout si quieres moverte.',
-    'pt_BR': 'Não se pode fazer reset no modo detached. Use checkout se quiser se mover',
+    'pt_BR': 'Não se pode fazer reset no modo detached. Use checkout se quiser mover',
```

Comment:
Here too I disagree. The message is referring to moving `HEAD`, which is kind of a "you are here" pointer that automatically follows you around as you move in the repository (from branch to branch, making commits, etc.). So indeed "se mover" is appropriate here.

Reply:
This is really a style issue, maybe the best would be: "Não é possível fazer reset no modo detached. Para mover-se, use checkout." I also checked the other translations in Spanish and they didn't use "si quieres si moverte".

Reply:
I wouldn't say it's a style issue — the verb "mover" is transitive, i.e. it requires an object; so ending with sentence with it and no object afterwards is incorrect grammar.

Here' what's being moved is one's position in the repository, hence the "se" in "mover-se". In Spanish they do the same thing, but they use the "te" in "moverte" to the same effect.

Reply:
ok I agree